webserver/views/api/v1/dataset_eval.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+from flask import Blueprint, jsonify, request
+from flask_login import current_user
+from webserver.decorators import auth_required
+import db.dataset_eval
+import db.dataset
+
+bp_dataset_eval = Blueprint('api_v1_dataset_eval', __name__)
+
+@bp_dataset_eval.route("/<uuid:job_id>/job-details", methods=["GET"])
+@auth_required
+def job_details(job_id):
+    """Returns the details of a particular job.
+       API key argument is required.
+
+       **Example Response**:
+
+       .. sourcecode:: json
+
+       {
+           "job_details": {
+               "created": "Tue, 07 Jun 2016 22:12:32 GMT",
+               "dataset_details": {
+                   "author": 1,
+                   "classes": [
+                       {
+                           "description": null,
+                           "id": "141",
+                           "name": "Class2",
+                           "recordings": [
+                               "d08ab44b-94c8-482b-a67f-a683a30fbe5a",
+                               "2618cb1d-8699-49df-93f7-a8afea6c914f"
+                           ]
+                       },
+                       {
+                           "description": null,
+                           "id": "142",
+                           "name": "Class1",
+                           "recordings": [
+                               "5251c17c-c161-4e73-8b1c-4231e8e39095",
+                               "c0dccd50-f9dc-476c-b1f1-84f00adeab51"
+                           ]
+                       }
+                   ],
+                   "created": "Mon, 02 May 2016 16:41:08 GMT",
+                   "description": "",
+                   "id": "5375e0ff-a6d0-44a3-bee1-05d46fbe6bd5",
+                   "last_edited": "Mon, 02 May 2016 16:41:08 GMT",
+                   "name": "test4",
+                   "public": true
+               },
+               "dataset_id": "5375e0ff-a6d0-44a3-bee1-05d46fbe6bd5",
+               "eval_location": "local",
+               "id": "7804abe5-58be-4c9c-a787-22b91d031489",
+               "options": {
+                   "filter_type": null,
+                   "normalize": false
+               },
+               "result": null,
+               "snapshot_id": "2d51df50-6b71-410e-bf9a-7e877fc9c6c0",
+               "status": "pending",
+               "status_msg": null,
+               "testing_snapshot": null,
+               "training_snapshot": null,
+               "updated": "Tue, 07 Jun 2016 22:12:32 GMT"
+           },
+          "success": true
+    }
+    """
+    job = db.dataset_eval.get_job(job_id)
+    job['dataset_details'] = db.dataset.get(job_details['dataset_id'])
+    return jsonify({'success' : True, 'job_details' : job})
```

Comment:
hmm. good point. I suggested we add it to the endpoint which returned a list, for consistency with the database endpoints, and because `jsonify` won't (wouldn't) serialise a top-level list. 
If we want to be consistent, what should we say here? `'status': 'ok'`?